proposed/0002-release-schedule.md

```diff
@@ -0,0 +1,118 @@
+  updates should wait for the next release (e.g. there should not be two MINOR releases in the same month)
+  - Users who need features immediately after they are merged should be encouraged to use either "dev-main" or "dev-next"
+    and not ask maintainers for quicker releases
+- PATCH fixes will *only* be back-ported to MAJOR versions that are less than 2 years (730 days) old - this is the sole
+  purpose of the "v*" branches
+  - Backporting to old releases can be done by a maintainer directly on the appropriate branches or by the community via a PR
+
+The "next" branch serves as a pre-release area for MAJOR changes in case there should be multiple MAJOR changes merged in
+a short period while allowing "main" to continue to be "stable" for users wanting only the latest MINOR updates. It is
+expected that most activity will occur in "main".
+
+#### Example
+
+As of now the current releases are 19.4.21 and 20.0.18. The next release containing only MINOR features would be 20.1.0, then 20.2.0
+until a MAJOR feature is released so 21.0.0 and so forth. If there is then a security patch it would be merged into "main" and "next"
+and released for the latest version (e.g. 21.0.1) and only backported to 20.2.1 (via a branch called v20) and 19.4.22 (via a branch called v19) in this case
+if those branches were less than two years old at the time.
+
+#### Effects
+
+- Relegates v19 to PATCH-only going forward and v19 basically becomes EOL in 2 years.
+- All new releases are at least MINOR versions so it is understood there is basically always a small upgrade risk. This resolves 
+  the common issue where a feature is BC breaking but in a very small way - this is always acceptable.
+- MINOR releases are as easy as tagging the latest "main" with a new release number. 
+- Bleeding-edge users and can easily use "dev-main" or "dev-next" via composer indefinitely.
+- Active users of tagged releases can upgrade as new releases are announced using the major/minor numbers as guidance as per semver standards.
+- Users wanting minimal upgrade hassle always have at least 2 years to get up to date to the latest major version for another two years or more.
+
+### Alternative 2
+
+We will follow the example of Ubuntu, which bases the Major Version Number on the Year.  
+And also has a specific Rythmus of an LTS version every 4 Releases,
+adding bigger experimantal features in the Major Release after an LTS version.
+
+But we will go with just one Major Release each year, which results in one LTS Release every 4 years.
+
+
+#### Visualization of the Ubuntu Release cycle
+
+![ubuntu release stragey visualized](https://github.com/OpenMage/rfcs/raw/main/assets/Ubuntu_release_cycle_Ubuntu.png?raw=true "ubuntu release strategy visualized")
+
+## Implementation
+
+- Vote on the alternatives above and choose the winner
+- Designate the appropriate "default" branch in GitHub admin
```

Comment:
this should already be in the RFC, otherwise it's not superseeding https://github.com/OpenMage/rfcs/pull/6 completely

Reply:
I updated for clarity